.circleci/config.yml

```diff
@@ -49,8 +49,10 @@ jobs:
       - rn/yarn_install:
           yarn_install_directory: examples/purchaseTesterTypescript
           cache_folder: ~/.cache/yarn
-      - rn/pod_install:
-          pod_install_directory: examples/purchaseTesterTypescript/ios
+      - run:
+          name: "Install example dependencies"
```

Comment:
Are quotes needed here? Most examples I've seen are without them

Reply:
Oh probably not 👍🏻

Reply:
```suggestion
          name: Install example dependencies
```